.ci/deps.go.sh

```diff
@@ -2,8 +2,7 @@
 
 set -ex
 
-go get -u github.com/alecthomas/gometalinter
+curl -L https://git.io/vp6lP | sh
```

Comment:
Did the old way work?  If so, keep your unbreaking change to only be the bit that unbreaks it.

What gometalinter encourages or discourages is irrlevant for our CI.  They encourage what is maintainable for them.  But we maintain our own CI, not them, and we dont beg them for help.

Also please rebase on top of https://github.com/coala/coala-bears/pull/2848 so we can see it working

Reply:
What will eventually happen is interesting, and guides our CI structure a bit, but the purpose of CI is to break.

However, when fixing a breakage, the commit *must* only do the things absolutely necessary to fix the breakage.  Then the commit describes the fix, not the fix and some other nice stuff.  In the future, someone will be wanting to understand why you made the changes you made, and it will take much longer to understand if they can not see a green build containing only the minimal changes needed to unbreak it.